src/magic.h

```diff
@@ -1,6 +1,6 @@
 /*************************************************************************/
-/*  Xania (M)ulti(U)ser(D)ungeon server source code                      */
-/*  (C) 1995-2000 Xania Development Team                                    */
+/*  Xania (Multi(User(Dungeon server source code                      */
```

Comment:
I think an errant global search/replace broke this?

Reply:
lol wtf...